ports/esp8266/esppwm.c

```diff
@@ -18,6 +18,7 @@
 
 #include "esppwm.h"
 
+#include "py/mphal.h"
```

Comment:
I don't think this include is needed anymore.

Reply:
removed